libs/abcvlib/src/main/java/jp/oist/abcvlib/fragments/PidGuiFragament.java

```diff
@@ -93,7 +94,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         controls.put("pw", p_wheel_ = rootView.findViewById(R.id.seekBarWheelSpeedP));
         controls.put("ew", expWeight_ = rootView.findViewById(R.id.seekBarExponentialWeight));
         controls.put("mt", maxAbsTilt_ = rootView.findViewById(R.id.seekBarMaxAbsTilt));
-
+        p_wheel_.setLabelFormatter(value -> String.format(Locale.US, "%.5f", value));
```

Comment:
I'm changing precision of hover values

Reply:
Were they just showing as 0.0 or something before? I don't recall this being an issue, but perhaps it has something to do with the theme change you made? I thought this was controlled by the `android:stepSize`